cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
+                        <% if (license.tooltip) { %>data-tooltip="<%- license.tooltip %>"<% } %>>
+                    <%- license.name %>
+                </button>
+                <p class="tip">
+                    <% if(license.url) { %>
+                        <a href="<%- license.url %>" target="_blank">
+                            <%= gettext("Learn more about {license_name}")
+                                        .replace("{license_name}", license.name)
+                            %>
+                        </a>
+                    <% } else { %>
+                        &nbsp;
+                    <% } %>
+                </p>
+            </li>
+        <% }) %>
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <label class="label setting-label" for="list-license-options">
+            <%- gettext("Options for {license_name}").replace("{license_name}", license.name) %>
+        </label>
+        <p class='tip tip-inline'>
+            <%- gettext("The following options are available for the {license_name} license.")
+                        .replace("{license_name}", license.name) %>
+        </p>
+        <ul class="license-options" id="list-license-options">
+            <% _.each(license.option_order, function(optionKey) { %>
+                <% var optionInfo = license.options[optionKey]; %>
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                            class="action-item license-button
+                                   <% if (optionSelected) { print("is-selected"); } %>
+                                   <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <i class="fa fa-fw
+                             <% if(optionSelected) { print("fa-check-square-o"); } else { print("fa-square-o"); } %>
+                             <% if(optionDisabled) { print("is-disabled"); } %>
+                             "></i>
+                        <h4 class="option-name"><%- optionInfo.name %></h4>
+                        <div class="explanation"><%- optionInfo.help %></div>
+                    </li>
+                <% } // could implement other types here %>
+            <% }) %>
+        </ul>
+    </div>
+<% } %>
+
+<% if (showPreview) { %>
+    <div class="license-preview-wrapper">
+        <label class="label setting-label" for="license-preview">
```

Comment:
```<label>``` elements are only allowed for use with form inputs.  This references a non-widget ```div``` element.  I think a Heading element is more appropriate here.  1 more than its parent (```<h3>``` -> ```<h4>```)